content/docs/how-we-work/git/git-pr.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Pull Request (PR)
+weight: 2
```

Comment:
@Yemaneberhan-Lemma, IIRC, the branching and merging had a weight of 4. It doesn’t make sense to talk about pull requests before talking about branching, no?